Readme.md

```diff
@@ -42,7 +42,7 @@ Html and css
   <summary>Client</summary>
   <ul>
     <li><a href="https://github.com/ghermaico135/Portfolio">portfolio</a></li>
-    <li><a href="https://ghermaico135.github.io/Portfolio/">portfolio</a></li>
+    <li><a href="https://ghermaico135.github.io/Portfolio/">online version portfolio</a></li>
```

Comment:
[Optional] I think that it would be more appropriate to include the link in the live demo section of the readme as contained in the [readme template](https://github.com/microverseinc/readme-template/blob/master/README.md).

![image](https://user-images.githubusercontent.com/20554583/230889151-fe0e2d69-0fab-4d5b-b037-5ff8f4d876c0.png)